discuss/overview/team_communication.rst

```diff
@@ -0,0 +1,117 @@
+======================================
+Efficiently Communicate Using Channels
+======================================
+
+You can use channels to organize discussions between individual teams, departments, projects, or any
+other group that requires regular communication. This way, you keep everyone in the loop updated
+with the latest developments.
+
+Public and Private channels
+===========================
+
+A *Public* channel can only be created by an administrator with write privileges and can be seen by
+everyone. By contrast, a *Private* one can be created by any user, and by default, is only visible
+to users invited to the channel.
+
+.. image:: media/create_channel.png
+   :align: center
+   :height: 350
+   :alt: View of discuss’s sidebar and a channel being created in Odo Discuss
+
+.. tip::
+   A public channel is best used when many employees need to access information (such as company
+   announcements), whereas a private channel could be used whenever information should be limited
+   to specific groups (such as a specific department).
+
+Configuration options
+---------------------
+
+You can configure a channel’s name, description, email alias, and privacy by clicking on the
+*Channel Settings* icon on the sidebar.
+
+.. image:: media/channel_settings.png
+   :align: center
+   :alt: View of a channel’s settings form in Odoo Discuss
+
+Privacy and Members
+~~~~~~~~~~~~~~~~~~~
+
+| Changing *Who can follow the group’s activities?* allows you to control which groups can have
+  access to the channel. Note that allowing *Everyone* to follow a private channel lets other users
+  view and join it, as they would a public one.
+| If you choose *Invited people only*, go to the *Members* tab to add your members, or, go to
+  Discuss’ main page, select the channel and click on *Invite*.
+
+.. image:: media/invite_channel.png
+   :align: center
+   :height: 300
+   :alt: View of Discuss’ sidebar emphasizing the option to invite members in Odoo Discuss
+
+For *Selected group of users*, the option *Auto Subscribe Groups* automatically add its members
```

Comment:
Use of bullet points? 
- Everyone
- Invited people only
- Selected group of users
It will bring more visibility. Because now, the text does not fit well.

Reply:
I am not explaining the auto-explanatory option *Everyone. The other two have their own explanation paragraphs. I will keep it as it is (: